core/src/blockchain/blockchain.rs

```diff
@@ -401,18 +401,14 @@ pub trait BlockProvider: HeaderProvider + BodyProvider + InvoiceProvider {
     }
 
     /// Get transaction with given transaction hash.
-    fn transaction(&self, address: &TransactionAddress) -> Option<LocalizedTransaction> {
```

Comment:
I realized that this commit has some problem.
I wanted to replace `UnverifiedTransaction` and `VerifiedTransaction` types to `validator::Transaction` type in this commit, but it was not applied. So I fixed that.
I also realized that `LocalizedTransaction` does not have to be removed.
But `TransactionView` type should be removed because we don't read fields of Txes in the host side.